### ISSUE TITLE
Enhancement: Additional Option to Set Idle Time Before Status Change to Invisible

### ISSUE THREAD

**Issue by angsaysroar** [RoguedBear/BetterDiscordPlugin-AutoIdleOnAFK#56]
It would be spectacular if you would consider adding an enhancement to this plugin that lets users set the amount of time they can be idle before their status is reverted to invisible.

I'm not sure how difficult this would be to implement, but I would love to see this feature added if at all possible.

**Comment by RoguedBear** [comment]
> that lets users set the amount of time they can be idle before their status is reverted to invisible.
just to clarify are you asking about a flow like this (see chart below) or something else?

```mermaid
flowchart TD
    A[user status online] -->|some time later| B(user minimises discord)
    B --->|5 minutes later| C[status set to idle]
    C --->|5 more minutes elapsed| D[status set to invisible]
```

**Comment by angsaysroar** [comment]
> > that lets users set the amount of time they can be idle before their status is reverted to invisible.
> 
> 
> 
> 
> 
> just to clarify are you asking about a flow like this (see chart below) or something else?
> 
> 
> 
> ```mermaid
> 
> flowchart TD
> 
>     A[user status online] -->|some time later| B(user minimises discord)
> 
>     B --->|5 minutes later| C[status set to idle]
> 
>     C --->|5 more minutes elapsed| D[status set to invisible]
> 
> ```
> 
>   

Yes, with the ability to specify the amount of time in the idle status before it switches to invisible.

**Comment by RoguedBear** [comment]
okay, i think issue #12 is what you're asking for.

I wont be able to commit a timeframe as i haven't been getting time to work on this plugin for a while now, but i'll have this feature prioritised when I do get around to rewriting this plugin.

if you can confirm #12 is what you're asking for that would help
